### DISCUSSION TITLE
Any plans for neo forge for this mod?

### DISCUSSION THREAD

**Discussion by Thunderrock424242** [Sinytra/Connector#128] [Q&A] [answered]
Since forge is kinda on a downhill 1.20.1 and fowwards

**Comment by Su5eD** [comment, marked as answer]
I agree, yes. I'm planning to migrate both FFAPI and Connector to NeoForge as soon as it becomes viable. Expectedly in 1.20.2 or the following mc release.

**Reply by Thunderrock424242** [reply]
Will you still update tho 1.20.1 with forge?

**Reply by coco875** [reply]
I think when 1.20.2 is out all mod stop maintening in 1.20.1 like for 1.19.2

**Reply by ChromaPIE** [reply]
@coco875 hello from the future. that's not the truth. and @Su5eD as this has been there for more than half a year, i wonder would the project be migrated to neoforge for 1.20.1? and is it possible for me to just brutally change the modloader to neoforge right now and still get everything working on the connector? (i guess not)

**Reply by coco875** [reply]
Connector already work for neoforge in 1.20.1 because the difference between forge and neoforge are very small. And it's was more a personal point of view if we see actually what modders have said and do is port to neoforge in 1.20.4 or wait 1.21

**Comment by purejosh** [comment]
There are breaking changes in Forge/Neoforge 1.20.2. At that point, you need to choose which to develop for. And there's no way Forge is going to survive this. Neoforge is the future. 

Congrats on finally getting into v1.0.0 beta for 1.20.1! That's a milestone for sure. Hopefully that means porting up to the new standards isn't too tough on you.
